test/module/shared_model/validators/container_validator_test.cpp

```diff
@@ -70,7 +70,7 @@ struct ContainerValidatorTest : public ::testing::Test {
  */
 TEST_F(ContainerValidatorTest, OldProposal) {
   shared_model::validation::DefaultProposalValidator validator(
-      iroha::test::kTestsValidatorsConfig);
+      iroha::test::kProposalTestsValidatorsConfig);
```

Comment:
Are changes required in this file? The proposals do not seem to contain duplicate transactions.

Reply:
Fixed